### PR TITLE
Typography token changes for V2

### PR DESCRIPTION
**Pull request by kalpeshp0310**

### CODE REVIEW

File: example/lib/styles/text_style.dart [hunk 1]
```diff
@@ -1,90 +1,74 @@
 import 'package:flutter/material.dart';
 
 /// TUITextStyle is a class that holds all the text styles used in the Tarka UI.
```
Comment (Ibrahimhass) [inline example/lib/styles/text_style.dart:3]: @kalpeshp0310 Please change the name in the comment as well.

Comment (kalpeshp0310) [reply example/lib/styles/text_style.dart:3]: I will remove the comments, as it is in example application and not really part of the library.

Comment (Ibrahimhass) [reply example/lib/styles/text_style.dart:3]: Agreed it is part of the kitchen-sink app.

Comment (kalpeshp0310) [reply example/lib/styles/text_style.dart:3]: Done.

**Review by Ibrahimhass — APPROVED** [review]

**Pull request merged**